geom/src/polygon.rs

```diff
@@ -120,8 +120,9 @@ impl Polygon {
 
     /// Does this polygon contain the point either in the interior or right on the border? Haven't
     /// tested carefully for polygons with holes.
+    // TODO Not sure about the "right on the border"
```

Comment:
Based on my reading of https://docs.rs/geo/latest/geo/algorithm/contains/trait.Contains.html, this only handles points completely in the interior. I don't believe anywhere else in the code actually relies on handling points directly on the boundary, so I just need to update the comment here

Reply:
That's right. You can use `intersects` if you want to include the boundary.

```
use geo::{point, Rect, Intersects, Contains};    
fn main() {    
    let border_point = point!(x: 1.0f64, y: 0.0);    
    let polygon = Rect::new(point!(x: 0.0f64, y: 0.0), point!(x: 2.0, y: 2.0)).to_polygon();    
    assert!(!polygon.contains(&border_point));    
    assert!(polygon.intersects(&border_point));    
} 
```

It's a common point of confusion, where "natural language" has a very specific meaning in this context.